app/services/reporting/account_deletion_rate_report.rb

```diff
@@ -0,0 +1,23 @@
+module Reporting
+  class AccountDeletionRateReport
+    attr_reader :report_date
+
+    def initialize(report_date = Time.zone.today)
+      @report_date = report_date
+    end
+
+    def account_deletion_report
+      table = []
+      table << ['Account deletion rate (last 30 days)']
+      table << [deleted_accounts_count]
```

Comment:
rate implies a percent: https://data.login.gov/account-deletions-report/

for data.login.gov we do it as a percent of signups that day, should we do the same here?

Reply:
and display in a table/row per week?

Reply:
the account deletions report uses the same exported data from `daily_registrations_report` to derive those rates

We should go ask our customers what format they want it in, I could see either one overall average for the last 30 days or having a week-by-week %